### ISSUE TITLE
Timing does not display Command Time

### ISSUE THREAD

**Issue by mattbreeden** [dbcli/pgcli#378]
My Pgcli install (0.19.1) always displays 0.000s for the Command Time. Format Time correctly displays times.

**Comment by amjith** [comment]
You are correct, sir. 

I think along the way we made pgcli efficient by not consuming the generator object returned by psycopg2 and pass it directly into the tabulate function which consumes the generator while building the table. 

I think it is time to get rid of the command time/format time split and merge it into a single value called 'time'. 

@dbcli/pgcli-core Thoughts?

**Comment by j-bennet** [comment]
I think I would rather merge it into a single line with both values present, like `Total time: 0.5sec (0.5s/0.1sec).` I think most people only care about query run time anyway, but you kind of want to catch those possible edge cases if pgcli suddenly takes inordinate amount of time formatting the results (tabulate has been iffy before).

**Comment by stuartquin** [comment]
I haven't had a thorough look at the code but I can't seem to make the Command Time return anything other than 0.000.
It's possible that my queries just aren't slow enough but I'm trying to make them as inefficient as possible!
In any case, I think it's still relevant to return an accurate command time, chances are the query will be used in an app somewhere so this is the value a developer will be concerned with

**Comment by j-bennet** [comment]
@stuartquin Fake it. :)

```
select schemaname, tablename from pg_tables, pg_sleep(3) where schemaname <> 'pg_catalog';
```

**Comment by darikg** [comment]
@stuartquin try adding `res = list(res)` after [this line](https://github.com/dbcli/pgcli/blob/54acaed24f4600f999d823568f13ca146fc60e1b/pgcli/main.py#L333) to force-exhaust the generator

**Comment by amjith** [comment]
We won't be able to separate the command time unless we exhaust the generator as suggested by Darik. 

Since `res` returned by this line: https://github.com/dbcli/pgcli/blob/54acaed24f4600f999d823568f13ca146fc60e1b/pgcli/main.py#L333
is a generator the actual command is not executed until we start consuming the generator. 

This line: https://github.com/dbcli/pgcli/blob/54acaed24f4600f999d823568f13ca146fc60e1b/pgcli/main.py#L357
takes in the cursor and inside the `format_output` method we start to consume the cursor at which point the command is actually executed and the results are formatted. 

So long story short, since we started passing around the cursor instead of a list of rows, we made pgcli memory efficient by choosing lazy evaluation. This means our format time is combined with the command time and we can only measure one time. No more separation. 

I'm going to create a PR to measure the time between this code block: https://github.com/dbcli/pgcli/blob/54acaed24f4600f999d823568f13ca146fc60e1b/pgcli/main.py#L331-L362
and call it total time. That's the best we can do, short of exhausting the cursor manually (which is inefficient).
